core/shared/utils/bh_log.h

```diff
@@ -73,6 +73,14 @@ bh_log(LogLevel log_level, const char *file, int line, const char *fmt, ...);
 void
 bh_print_time(const char *prompt);
 
+void
+bh_print_vm(const char *prompt);
+
+void
+bh_log_vm(const char *function, uint32 line);
```

Comment:
How about renaming them to `bh_print_vm_mem` and `bh_log_log_vm_mem`, and renaming `LOG_MEM` to `LOG_VM_MEM`? They are more understandable.

Reply:
`vm` is for *Virtual memory* here. So, maybe `bh_print_mem` and `bh_log_mem` ?

Reply:
vm makes a little confused, it may refer to `virtual machine`, how about `bh_print_virtual_mem` and `bh_log_virtual_mem` to make it clear?